CONTRIBUTING.md

```diff
@@ -1,5 +1,47 @@
 # How to contribute
 
-If you encounter any bugs with the library please file an issue in the [Issues](https://github.com/Azure/azure-sdk-for-java/issues) section of the project.
+Thank you for your interest in contributing to Azure SDK for Java.
 
-If you would like to become an active contributor to this project please follow the instructions provided in [Microsoft Azure Projects Contribution Guidelines](http://azure.github.com/guidelines.html).
\ No newline at end of file
+# Ways to contribute
+- If you encounter any bugs with the library please file an issue in the [issues](https://github.com/Azure/azure-sdk-for-java/issues) section of the project.
+
+
+## Getting started
+If you would like to become an active contributor to this project please follow the instructions provided in [Microsoft Azure Projects Contribution Guidelines](http://azure.github.com/guidelines.html).
+
+If you wish to make code changes, or contribute something new, please follow the [GitHub Forks / Pull requests model](https://help.github.com/articles/fork-a-repo/): Fork the repo, make the change and propose it back by submitting a pull request.
+
+
+## Setting up a developer environment
+- Install Java Development Kit 8
```

Comment:
add a link

Reply:
I removed the link as per @JonathanGiles  suggestion that we shouldn't tell people to use Oracle Java or OpenJDK , but just ask for JDK8.
Makes sense I think